source/lmp/fix_dplr.cpp

```diff
@@ -142,18 +138,70 @@ FixDPLR::FixDPLR(LAMMPS *lmp, int narg, char **arg)
     error->one(FLERR, e.what());
   }
 
+  pair_deepmd = (PairDeepMD *)force->pair_match("deepmd", 1);
+  if (!pair_deepmd) {
+    error->all(FLERR, "pair_style deepmd should be set before this fix\n");
+  }
+
+  int n = atom->ntypes;
+  std::vector<std::string> type_names = pair_deepmd->type_names;
+  std::vector<std::string> type_map;
+  std::string type_map_str;
+  dpt.get_type_map(type_map_str);
+  // convert the string to a vector of strings
+  std::istringstream iss(type_map_str);
+  std::string type_name;
+  while (iss >> type_name) {
+    type_map.push_back(type_name);
+  }
+  if (type_names.size() == 0 || type_map.size() == 0) {
```

Comment:
As I understand, type_map comes from the type map stored in the DeepTensor model. Is it possible that the DeepTensor model stores no information about type map so that type_map.size()==0?

Reply:
If no type map is stored, it's expected to fall back to the old behavior, i.e. line 158-161.

Reply:
I see.